src/showcase/src/pages/components/ChoosePin.astro

```diff
@@ -0,0 +1,63 @@
+---
+import "$src/styles/main.css";
+---
+
+<div>
+  <choose-pin></choose-pin>
+</div>
+
+<script>
+  import { LitComponent } from "./lit";
+
+  import { pinInput } from "$src/components/pinInput";
+  import { Chan } from "$src/utils/utils";
+  import { html } from "lit-html";
+  import { asyncReplace } from "lit-html/directives/async-replace.js";
+
+  class ChoosePin extends LitComponent {
```

Comment:
This LitComponent is closed per default - i.e. is does not generate a true WebComponents right? Just asking in case that would prevent the propagation of CSS and therefore lead to unexpected styling issue

Reply:
Can you clarify what you mean by "closed per default"? I don't have much experience with webcomponents.

In this case there's no shadow DOM and we really do expect the out CSS to be applied

Reply:
> The {mode: "open"} argument gives the page a way to break the encapsulation of your shadow DOM. If you don't want to give the page this ability, pass {mode: "closed"} instead, and then shadowRoot returns null.

https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM

If "closed", then the component is a WebComponent and only the CSS within the component style the component, defining global CSS rules have no effect on inside the component. "open" the contrary.


// <my-component>
//    #shadow-root (closed)
//    <div style="bg: green">I'm green 
//
// <my-other-component>
//    (open)
//    <div style="bg: green">I'm red
//
// div.css 
//    bg: red

Reply:
Gotcha thanks! though as mentioned, they don't even use a Shadow DOM and the goal is for the outer CSS to be applied. Everything ok then right?

<img width="606" alt="Screenshot 2023-11-02 at 15 16 27" src="https://github.com/dfinity/internet-identity/assets/6930756/49d98b1a-cc42-4427-88c8-70de7044d83e">

Reply:
Looks good! So no WebComponents according the book but, custom elements. Maybe was just my misunderstanding, thanks for the feedbacks.

Reply:
Ah no my bad; custom elements is definitely the right (or at least more correct) term!